src/app/dataset-block/metadata-block/components/event-details/components/common/owner-property/owner-property.component.html

```diff
@@ -0,0 +1 @@
+<span>{{ data.name }}</span>
```

Comment:
This component should be a link to the user page.

Reply:
Changed to link